api/routes/user.js

```diff
@@ -276,11 +275,11 @@ router.post('/edit', (req, res) => {
     if (result.nModified < 1) {
       return res
         .status(NOT_FOUND)
-        .send({ message: `${req.body.queryEmail} not found.` })
+        .send({ message: `${req.body.email} not found.` })
```

Comment:
for improved readability
```suggestion
        .send({ message: `${query.email} not found.` })
```